### DISCUSSION TITLE
Change image width

### DISCUSSION THREAD

**Discussion by gcailly** [jpanther/congo#323] [Q&A]
Hi,
I would like to change the image width.
I some times use small images but they are enlarged and become pixelated.
In the docs I found the figure shortcode but no direct option for width.
I added `default=true width=50%` : it works but all the benefits you added in your shorcode is lost.
Rounded corners are lost too, which is a shame.
Do you know how I could do ?

**Comment by jneidel** [comment]
One way changing width can be achieved without changing any code is by setting it in the class attribute:

```
{{<figure class="w-9/12 mx-auto my-0 rounded-md" src="pic.jpg">}}
```

- [ref tailwind width values](https://tailwindcss.com/docs/width)
- requires [tailwind jit compiler to be running](https://jpanther.github.io/congo/docs/advanced-customisation/#building-the-theme-css-from-source)

**Comment by stereobooster** [comment]
Yes, images smaller than `660px` still resized

```
        {{ else }}
          src="{{ (.Resize "660x").RelPermalink }}"
          srcset=" {{- (.Resize "330x").RelPermalink }} 330w,
          {{- (.Resize "660x").RelPermalink }} 660w
          {{ if gt .Width 1024 }}
            ,{{ (.Resize "1024x").RelPermalink }} 1024w
          {{ else }}
```

theoretically we can add check to make sure to not resize images smaller than 660, 330px. Or as well we can add option to opt-out from resizing at all
